sunpy/database/database.py

```diff
@@ -388,7 +388,21 @@ def _download_and_collect_entries(self, query_result, **kwargs):
         if client is None:
             client = VSOClient()
 
-        paths = client.get(query_result, path, methods).wait(progress=progress)
+        remove_list = []
+        for qr in query_result:
+            temp =  tables.DatabaseEntry._from_query_result_block(qr)
+            for database_entry in self:
+                if database_entry.path is not None and temp._compare_attributes(
+                    database_entry, ["source", "provider", "physobs", "fileid", 
+                    "observation_time_start", "observation_time_end", 
+                    "instrument", "size", "wavemin", "wavemax"]):
+                    remove_list.append(qr)
+                    break
+
+        for temp in remove_list:
+            query_result.remove(temp)
+
+        paths = client.get(query_result, path).wait(progress=progress)
```

Comment:
Was it intentional to remove `methods` as an argument to `get()` (see the old line 391)?  Now, `methods` is no longer used at all in this function.